app/services/research_trends_report_generator.rb

```diff
@@ -79,18 +84,26 @@ def generate_crosstab_row(concept, concept_years, years, total)
   # @return [ActiveRecord::Result]
   def database_values
     conn = ActiveRecord::Base.connection
-    conn.exec_query(sql, 'SQL', [[nil, Person.org_metadata_field(organization.type)], [nil, organization.uri]])
+    conn.exec_query(sql, 'SQL', [[nil, Person.org_metadata_field(organization.type)],
+                                 [nil, organization.uri],
+                                 [nil, start_year],
+                                 [nil, end_year]])
   end
 
+  # rubocop:disable Metrics/MethodLength
   def sql
     'SELECT con.name as concept, dpub.created_year as year, count(*) ' \
     "FROM (SELECT DISTINCT pub.uri, pub.metadata -> 'created_year' as created_year, " \
     "jsonb_array_elements_text(pub.metadata -> 'concepts') as concept_uri FROM publications pub " \
     'INNER JOIN people_publications pp on pub.uri=pp.publication_uri ' \
     'INNER JOIN people p on p.uri=pp.person_uri ' \
-    'WHERE p.metadata -> $1 ? $2) as dpub ' \
+    'WHERE p.metadata -> $1 ? $2 '\
+    "AND pub.metadata -> 'created_year' >= $3 AND " \
+    "pub.metadata -> 'created_year' <= $4 " \
```

Comment:
I have a minor preference that the `AND ` move from the prior line to the beginning of this string, for consistency w/ the lines above, but it's no big.